.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+## Description
+
+<!-- add description here -->
+
+## Checklist
+
+- [ ] I have added or updated any relevant documentation.
+- [ ] I have cleaned any remaining cloud resources from my accounts.
```

Comment:
Is this last part required? Seems quite out of place considering these are all public repos.

Also, on checklist, if we add documentation check, we should also add `I have added or updated relevant tests`. Not all PRs require docs, and not all PRs require tests, so the same rationale would apply